### ISSUE TITLE
When evaluating with map/ap, result is all 0

### ISSUE THREAD

**Issue by zf6578** [open-mmlab/mmpose#3142]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmpose/issues) and [Discussions](https://github.com/open-mmlab/mmpose/discussions) but cannot get the expected help.
- [X] The bug has not been fixed in the latest version(https://github.com/open-mmlab/mmpose).

### Environment

hi,
when i  evaluating my model with metric map/ap , it comes:
![image](https://github.com/user-attachments/assets/0427d6c9-916d-4c09-8ff6-3febbd1b6104)
![image](https://github.com/user-attachments/assets/2cd82db0-d7c3-404b-a787-ebd003b09ba9)
The overall loss function is decreasing, acc_pose is increasing, but map remains at 0
Then I visualized the results of the model inference and found that the results were still good. So, where did the problem lie

### Reproduces the problem - code sample

python train.py

### Reproduces the problem - command or script

python train.py

### Reproduces the problem - error message

![image](https://github.com/user-attachments/assets/65b03d3e-8ab9-4aa9-a2a6-874a774ebe74)

**Comment by Liang-HHHHH** [comment]
I encountered a similar problem, but it can be solved by reducing the learning rate  ：base_lr = 0.0001
